src/Makefile.am

```diff
@@ -207,7 +207,8 @@ GRIDCOIN_CORE_H = \
     wallet/generated_type.h \
     wallet/walletdb.h \
     wallet/wallet.h \
-    wallet/ismine.h
+    wallet/ismine.h \
+	wallet/diagnose.h
```

Comment:
Please keep spacing consistent.

Reply:
It is vim that adding them :)
I have fixed the indentation in my next commit